samples/istio/service-to-service/README.md

```diff
@@ -0,0 +1,198 @@
+# Service to Service - East-West Traffic
+
+In this scenario, we deploy Choreo Connect within the Istio service mesh where it applies API Management for microservices to microservice communication.
```

Comment:
In this scenario, we deploy Choreo Connect as an Internal API Gateway within the Istio service mesh, where it applies API Management to microservices communication.